src/utils.ts

```diff
@@ -83,6 +47,15 @@ export function createHeaders(options: Partial<Pick<RequestOptions, "headers" |
   return headers;
 }
 
+export function isHttpieError(error: unknown): error is HttpieError {
+  return error instanceof HttpieError;
+}
+
+// eslint-disable-next-line max-len
+export function isHTTPError<T extends RequestResponse<any> = RequestResponse<any>>(error: unknown): error is HttpieOnHttpError<T> {
```

Comment:
```suggestion
export function isHTTPError<T extends RequestResponse<any> = RequestResponse<any>>(
 error: unknown
): error is HttpieOnHttpError<T> {
```

Reply:
Also, note that you wrote `error: any` instead of `error: unknown` is the doc example, is it intended ?